packages/nouns-contracts/contracts/governance/NounsDAOV3Admin.sol

```diff
@@ -575,4 +579,16 @@ library NounsDAOV3Admin {
         require(n <= type(uint32).max, errorMessage);
         return uint32(n);
     }
+
+    function checkForDuplicates(address[] calldata erc20tokens) internal pure {
+        for (uint256 i = 0; i < erc20tokens.length; i++) {
+            if (erc20tokens[i] == address(0)) revert TokenCantBeAddressZero();
+
+            if (i < erc20tokens.length - 1) {
```

Comment:
@eladmallel instead of this, you can make the for loop only run until `erc20tokens.length - 1`:

`for (uint256 i = 0; i < erc20tokens.length - 1; i++) {`

Reply:
@davidbrai I kept it running all the way bc I wanted to check for address zero on the last token too. I could do that outside of the loop, but instead chose this style. Are you suggesting:

1. drop zero checks altogether
2. check outside the loop
3. skip the last item only

?

Reply:
zero check doesn't really make sense in a `checkForDuplicates` function
I'm ok with dropping the zero check altogether